lib/widgets/tile_action.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+import '../hooks/delayed_loading.dart';
+
+class TileAction extends StatelessWidget {
```

Comment:
not really, because it doesn't have size constraints as `TileAction` has so it would require a bit of reworking. perhaps we could pass some kind of enum that would specify the compactness of the button or sth like that